src/main/java/com/farao_community/farao/gridcapa/job_launcher/JobLauncherService.java

```diff
@@ -14,7 +14,7 @@
  */
 @Service
 public class JobLauncherService {
-    private static final Logger LOGGER = LoggerFactory.getLogger(JobLauncherService.class);
+    private static final Logger BUSINESS_LOGGER = LoggerFactory.getLogger("BUSINESS_LOGGER");
```

Comment:
Isn't it possible to define this logger as a bean for the whole application ? Maybe not useful in job-launcher case but it might be useful later

Reply:
I agree with the idea, to extract that aside, but not using a bean. 
I would use a class that could be instantiated only if given a task id, that we use to set it as extra field in MDC.
What do you think?

Reply:
ok, no risk if we instantiate this class several times ? In different classes for example ?